apps/dataflow/src/scraper/fms/driver.py

```
@@ -0,0 +1,203 @@
+    for idx in range(len(floor_links)):
+        current_floor_links = driver.find_elements(
+            By.XPATH,
+            f"//a[contains(normalize-space(.), '{building_label}')]/ancestor::div[1]/following-sibling::ul"
+            "//div[contains(@class,'fmi-nav-floorplan')]/a[contains(@class,'rtIn')]"
+        )
+
+        link = current_floor_links[idx]
+        href = link.get_attribute("href")
+        label = link.text.strip()
+
+        print(f"Opening floor {label} ({idx+1}/{len(floor_links)}) -> {href}")
```

Ruff (0.14.14)

[warning] `print` found

Remove `print`

(T201)